crates/varcon-core/fuzz/fuzz_targets/cluster_parse.rs

```diff
@@ -0,0 +1,6 @@
+#![no_main]
```

Comment:
This seems to be part of varcon and not typos.

Reply:
while varcon is its own crate, the upstream is this repo: https://crates.io/crates/varcon-core